src/registrar/tests/services/test_dns_host_service.py

```diff
@@ -1,7 +1,12 @@
 from unittest.mock import patch, Mock
-from django.test import SimpleTestCase
+from django.test import SimpleTestCase, TestCase
+from django.db import IntegrityError
 
 from registrar.services.dns_host_service import DnsHostService
+from registrar.models.dns.dns_account import DnsAccount
+from registrar.models.dns.vendor_dns_account import VendorDnsAccount
+from registrar.models.dns.dns_account_vendor_dns_account import DnsAccount_VendorDnsAccount as Join
```

Comment:
```suggestion
from registrar.models.dns.dns_account_vendor_dns_account import DnsAccount_VendorDnsAccount as AccountsJoin
```

Suggestion to be more specific here- we will also have joins for Zone and DnsRecord to reference.